src/compile/common.ts

```diff
@@ -103,7 +103,9 @@ export function formatMixins(model: Model, channel: Channel, format: string) {
   let def: any = {};
 
   if (fieldDef.type === TEMPORAL) {
```

Comment:
The current code seems to output `formatType` no matter if it outputs `format` or not.